airrohr-firmware/airrohr-firmware.ino

```diff
@@ -3457,6 +3468,48 @@ void fetchSensorSPH0645(String& s){
 
 }
 
+/*****************************************************************
+ * Initialize RTC                                        		 *
+ *****************************************************************/
+void init_RTC()
+{
+	pinMode(RTC_PIN_SDA, OUTPUT);
+	pinMode(RTC_PIN_SCL, OUTPUT);
+	if (!rtc.isrunning())
+	{
+		// sets time to the date this sketch was compiled
+		// rtc.adjust(DateTime(F(__DATE__), F(__TIME__)));
+		// Set explicit time for example 19th March 2020 at 12 noon.
+		// rtc.adjust(DateTime(2020, 5, 19, 12, 00, 00));
+	}
+}
+
+/*****************************************************************
+ * read RTC sensor values                                        *
+*****************************************************************/
+static void fetchSensorRTC(String &s) {
```

Comment:
@Warukira why do we need a function that only prints out date values?

Reply:
@chegejames , we could add more functionality. This one gets the current time and prints it out.

Reply:
@Warukira we want to a timestamp with each sensor reading.